src/software/thunderscope/robot_diagnostics/robot_info.py

```diff
@@ -121,7 +121,16 @@ def __init__(
         self.primitive_loss_rate_label = common_widgets.ColorQLabel(
             max_val=MAX_ACCEPTABLE_PACKET_LOSS_PERCENT
         )
+
+        # Primitive round-trip time label and queue
+        self.primitive_rtt_label = common_widgets.ColorQLabel(
+            max_val=MAX_ACCEPTABLE_MILLISECOND_ROUND_TRIP_TIME,
+            text="MS"
```

Comment:
Lets make the default value 
```suggestion
            text="RTT:NA"
```